chapter_multilayer-perceptrons/numerical-stability-and-init_vn.md

```diff
@@ -321,7 +331,10 @@ Perhaps you may come across a recent clever idea and contribute its implementati
 -->
 
 *dịch đoạn phía trên*
-
+1. Bạn có thể thiết kế các trường hợp phá vỡ đối xứng khác bên cạnh đối xứng hoán vị?
+2. Chúng ta có thể khởi tạo tất cả trọng số ở trong mạng hồi quy tuyến tính hoặc trong hồi quy softmax cùng một giá trị hay không?
+3. Tra cứu phân tích về giới hạn trong trị riêng của phép nhân hai ma trận. Điều này cho bạn biết gì về việc đảm bảo gradients được điều hòa tốt?
+4. Nếu như chúng ta biết rằng một số số hạng phân kỳ (quá trình training) thì chúng ta có thể sửa được không? Nhìn vào bài báo về LARS để tìm cảm hứng :cite:`You.Gitman.Ginsburg.2017`.
```

Comment:
4. Nếu biết rằng trong vài trường hợp, mô hình không thể hội tụ, liệu ta có thể sửa chữa vấn đề này (mô hình bị phân kỳ do vanishing/exploding gradient). Có thể xem gợi ý ở paper LARS :cite:`You.Gitman.Ginsburg.2017`.